resources/core/charts/kubeless/charts/lambdas-ui/README.md

```diff
@@ -0,0 +1,119 @@
+# Lambdas
+
+## Overview
+
+**Lambdas** is a web-based UI component which is a part of the Console UI.
+It allows you to manage lambda functions, which are the key part of Kyma capabilities.
+
+## Details
+
+**Lambdas** allow you to:
+- Create and modify lambda functions.
+- Expose a function as API.
+- Bind a function to a service.
+- Write the code of the function. This option is available only through node.js.
+- Manage dependencies.
+- Add and remove labels.
+- Enable API security.
+- Select the function size.
+
+### Configuration
+
+**Lambdas** UI is deployed with the use of a [ConfigMap](templates/configmap.yaml).
+The **ConfigMap**  includes the `config.js` file that is mounted as the asset of the Console application and injected as a configuration file.
+Use this mechanism to overwrite the default configuration with custom values resulting from the Helm chart installation.
+
+### Function size
+
+You can modify the function size when you either create or update a function using  **Lambdas** UI.
+
+The [ConfigMap](templates/configmap.yaml) contains a set of configurations for a function size, including
+- **resources:** configuration with the following parameters: **requests:** and **limits:**.  
```

Comment:
```suggestion
- **resources** configuration with the following parameters: **requests** and **limits**.  
```

I removed the colons, because actually when we talk about general configuration and the options users have there, I would use just the names of configuration elements - the colons may be a bit confusing because sometimes they appear to be a part of the sentence.